crowbar_framework/app/models/network_service.rb

```diff
@@ -413,8 +412,6 @@ def build_net_info(network, name, db = nil)
     db["network"].each { |k,v|
       net_info[k] = v unless v.nil?
     }
-    net_info["usage"]= network
-    net_info["node"] = name
```

Comment:
Really, which one? Looking at https://github.com/crowbar/crowbar-core/pull/510/files#diff-0998300bd8296b99808ceb69e36a90d8R406
it's still there ```def build_net_info(network, name, db = nil)```